src/PlusWidgets/CMakeLists.txt

```diff
@@ -45,16 +45,14 @@ SET(${PROJECT_NAME}_INCLUDE_DIRS
 IF(NOT (CMAKE_VERSION VERSION_LESS "3.8"))
   # CMake 3.8 and later puts generated headers (such as ui_QPlusDeviceSetSelectorWidget.h) into
   # .../PlusLib-bin/src/PlusWidgets/PlusWidgets_autogen/include
-  IF(IS_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/${PROJECT_NAME}_autogen/include)
-    LIST(APPEND ${PROJECT_NAME}_INCLUDE_DIRS ${CMAKE_CURRENT_BINARY_DIR}/${PROJECT_NAME}_autogen/include)
-  ENDIF()
-  
+  file(MAKE_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/${PROJECT_NAME}_autogen/include)

```

Comment:
Random question, is the make_directory necessary?

Reply:
I tried this without it - and if the directory doesn't exist CMake will fail with the error "include directories not found".